index.js

```diff
@@ -27,6 +27,9 @@ function fail (err) {
 
 function openConfig (cb) {
   var userHome = require('user-home');
+  if (process.platform === 'linux' && userHome) {
+    userHome += '/.cache';
```

Comment:
I think you should scope this by if the platform is windows or not. OSX has a standard `~/.cache` path as well.  Also, if windows has a similar path, we should use it (and use `path.join` here as well).